package.json

```diff
@@ -33,6 +33,7 @@
     "babel-eslint": "^6.1.2",
     "babel-jest": "^15.0.0",
     "babel-loader": "^6.2.5",
+    "babel-plugin-rewire": "^1.0.0",
```

Comment:
@vutran This PR now includes `babel-plugin-rewire`, 
which I use to [mock the `fs` module](https://github.com/adnasa/dext/blob/6bda0795398987327d1f32adcad0aecc78af949b/__tests__/app/main/plugins.js#L163-L171)